Library/Formula/juman.rb

```diff
@@ -0,0 +1,15 @@
+class Juman < Formula
+  homepage "http://nlp.ist.i.kyoto-u.ac.jp/index.php?JUMAN"
+  url "http://nlp.ist.i.kyoto-u.ac.jp/nl-resource/juman/juman-7.01.tar.bz2"
+  sha1 "e22eb113f40216b8a04d811484a94f93f834ae9a"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "juman", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!